fpa/minerselector/fixed/fixed.go

```diff
@@ -0,0 +1,33 @@
+package fixed
```

Comment:
The `fpa/minerselector` concept are possible implementations of miner-selection algorithms.

This `fpa/minerselector/fixed` is an implementation that always return a fixed miner.